zebra-chain/src/transaction.rs

```diff
@@ -160,6 +162,22 @@ impl Transaction {
         sighash::SigHasher::new(self, hash_type, network_upgrade, input).sighash()
     }
 
+    /// Compute the authorizing data commitment of this transaction as specified
+    /// in [ZIP-244].
+    ///
+    /// Returns None for pre-v5 transactions.
+    ///
+    /// [ZIP-244]: https://zips.z.cash/zip-0244.
+    pub fn auth_digest(&self) -> Option<AuthDigest> {
+        match self {
+            Transaction::V1 { .. }
+            | Transaction::V2 { .. }
+            | Transaction::V3 { .. }
+            | Transaction::V4 { .. } => None,
```

Comment:
So, I know in [ZIP-244](https://zips.z.cash/zip-0244#authorizing-data-commitment), it says that for <= V4, it says that, `a placeholder value consisting of 32 bytes of 0xFF is used in place of the authorizing data commitment. This is only used in the tree committed to by hashAuthDataRoot`. Should that placeholder value be returned here? @teor2345

Reply:
We need to use the `auth_digest` as a field in wide transaction IDs (see ticket #2449).

So I'll need a method that returns `Option<AuthDigest>`.
But feel free to create a wrapper method that substitutes the placeholder for `None`.